site/docs/reference/Connectors/materialization-connectors/pinecone.md

```diff
@@ -25,30 +25,32 @@ To use this connector, you'll need:
 
 ## Embedding Input
 
-The materialization creates vector embeddings based on a text input from your collection. A field
-with type `string` must exist in your collection and it must be `required` in the collection schema.
+The materialization creates vector embeddings based on fields in your collection.
 
-By default the materialization connector will look for a field named `"input"` in your collection
-and use that value to create the embeddings. If you are using a
-[derivation](../../../concepts/derivations.md) to transform your source data before materializing it
-into Pinecone it may be convenient to create the derived collection with this field. Alternatively a
-[projection](../../../concepts/advanced/projections.md) can be configured for the source collection
-with the name of `"input"`. There is also an optional configuration for each binding to set an
-alternate name of the collection projection to use as the embedding input (see Bindings below).
+By default, fields of a single scalar type are including in the embedding: strings, integers,
+numbers, and booleans. Additional array or object type fields can optionally be included using
```

Comment:
Stylistic nit, but FYI, it's a good idea to use active voice and refer to user as "you" whenever it feels natural. So here, saying "You can include additional array or object type fields using projected fields."